scripts/dump-co-pilot-reporting-data.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can you use `#!/usr/bin/env bash` here?  It's a little more portable and how we do it elsewhere.

Reply:
I can, but each of the other scripts in `scripts` uses `#!/bin/bash` rather than `#!/usr/bin/env bash`. I agree that the latter is more portable. Should I change them all?

Reply:
Oh yes, that would be great.  IIRC @madwort normalised all our justfile bash invocations to the env-based one, so it would be good to be consistent.

Reply:
I've changed them all.